system/types/app_settings.go

```diff
@@ -50,6 +50,12 @@ type (
 				// Can users reset their passwords
 				PasswordReset struct{ Enabled bool } `kv:"password-reset"`
 
+				// Can users create their passwords
```

Comment:
_This little thing did not get submitted when I reviewed this yesterday._

Who creates passwords for whom? This comment does not really describes it right...